src/controllers/employee.js

```
@@ -1,180 +1,606 @@
     }
+  } catch (error) {
+    console.log(error);
   }
+};
 
+exports.showManagers = async (req, res) => {
+  const data = await db.employee.aggregate("managerId", "DISTINCT", {
+    plain: false,
+  });
+  const ids = [];
+  for (let d of data) {
```

Biome

[error] This let declares a variable that is only assigned once.

'd' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)